Registry/registry.fire

```diff
@@ -67,6 +67,31 @@ state    real     burnt_area_dt *i*j      fire        1         z    hr    "BURN
 state    real     flame_length  *i*j      fire        1         z    hr    "FLAME_LENGTH"     "fire flame length" "m"
 state    real     ros_front     *i*j      fire        1         z    hr    "ROS_FRONT"     "rate of spread at fire front" "m/s"
 
+# fuel moisture variables
+state    real     fmc_g     *i*j      fire        1         z   i102hr "FMC_G"     "ground fuel moisture contents" "1"
+
+# fuel moisture model section
+dimspec  num_fmc    -   namelist=nfmc              z      fuel_moisture_classes 
+rconfig  integer  nfmc  namelist,fire  1    5  -   "nfmc" "number of fuel moisture classes"
```

Comment:
@janmandel 
Jan,
Where are these new variables packaged? If not, they should be.

Reply:
Dave,
all state variables in registry.fire are now packaged.
Jan